dll/win32/browseui/precomp.h

```diff
@@ -6,6 +6,8 @@
 #define WIN32_NO_STATUS
 #define _INC_WINDOWS
 #define COM_NO_WINDOWS_H
+#define BWM_SETTINGCHANGE (WM_USER + 300)
+#define BWM_GETSETTINGS (WM_USER + 301)
```

Comment:
Can you move those two defines together with the struct BrowseUISettings definition below?